crowdgezwitscher/crowdgezwitscher/settings/default.py

```diff
@@ -125,7 +125,18 @@
 
 # This is the default key, for which all verification requests will pass.
 RECAPTCHA_SECRET_KEY = ['6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe']
+######################################################################################################
+
+
+######################################################################################################
+#                                           Twitter settings
+######################################################################################################
+# number of Tweets to request from Twitter's REST API
+TWITTER_TWEET_COUNT = 50
 
+# replace or override with sane values
+TWITTER_CONSUMER_KEY = ''
+TWITTER_CONSUMER_SECRET = ''  # noqa
```

Comment:
Ahh ich find das nicht so schön. Das geht bestimmt mal schief. Können wir nicht auch die `sensitive_data.py` in `settings/dev.py` importieren?